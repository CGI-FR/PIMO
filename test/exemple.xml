<?xml version="1.0" encoding="UTF-8"?>
<taxes>
    <agency>
        <name>Nantes Agency</name>
        <agency_number>0032</agency_number>
    </agency>
    <account>
        <name>Doe John</name>
        <account_number>12345</account_number>
        <annual_income>50000</annual_income>
        <total_taxes_paid>10000</total_taxes_paid>
    </account>
    <account>
        <name>Smith Jane</name>
        <account_number>67890</account_number>
        <annual_income>60000</annual_income>
        <total_taxes_paid>12000</total_taxes_paid>
    </account>
</taxes>
