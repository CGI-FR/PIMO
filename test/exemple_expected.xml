<?xml version="1.0" encoding="UTF-8"?>
<taxes>
    <agency>
  <name>Nantes Agency</name>
  <agency_number>2308</agency_number>
</agency>
    <account>
  <name>Rolande</name>
  <account_number>1</account_number>
  <annual_income>50000</annual_income>
  <total_taxes_paid>10000</total_taxes_paid>
</account>
    <account>
  <name>Mat√©o</name>
  <account_number>2</account_number>
  <annual_income>60000</annual_income>
  <total_taxes_paid>12000</total_taxes_paid>
</account>
    <account>
  <name>Rosalie</name>
  <account_number>3</account_number>
  <annual_income>60000</annual_income>
  <total_taxes_paid>12000</total_taxes_paid>
</account>
</taxes>
